app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -412,13 +397,7 @@ function fetchNextPageIfNearBottom() {
     };
   }
 
-  var scrollableElem = document.getElementById(scrollableElemId);
-
-  if (
-    !done &&
-    !fetching &&
-    window.scrollY > scrollableElem.scrollHeight - 3700
```

Comment:
I'm not sure what the context was around this `3700px` value - it's been around since the initial commit.

I've added a 500px margin to the `IntersectionObserver` so that new content loads when the user scrolls to within 500px of the page footer.

Reply:
We probably need to adjust this amount of pixels depending on the device. This fine for desktop, but on smaller mobile devices it might be called too frequently. @ben, any idea what the `3700` is supposed to represent?